services/httpd/handler.go

```diff
@@ -285,6 +290,15 @@ func (h *Handler) AddRoutes(routes ...Route) {
 			handler = http.HandlerFunc(hf)
 		}
 
+		// Throttle route if this is a write endpoint.
+		if r.Method == "POST" {
```

Comment:
Prefer `http.MethodPost` but not important

Reply:
Fixed in 8a74c67.